### DISCUSSION TITLE
Where to report docset issues?

### DISCUSSION THREAD

**Discussion by ferenczy** [zealdocs/zeal#1519] [Q&A]
Just a quick question: **where can I report the docset issues**, please?

I have found an issue in the _.NET Framework_ docset on Ubuntu 22.04 - it doesn't display the docset home page, because there's an invalid path to that page in the file `NET_Framework.docset/Contents/Info.plist`.

Invalid path:
`msdn.microsoft.com/en-US/library/mt472912(v=vs.110).html`

Correct path:
`msdn.microsoft.com/en-us/library/mt472912(v=vs.110).html`
(the path is all lowercase including the language code in the file system)

Thank you.

**Comment by trollixx** [comment]
This is not really an issue with docset as it works on Windows and macOS due to case-insensitive file system. More similar issues are reported in https://github.com/zealdocs/zeal/issues/1009.
